commands/inspect.go

```diff
@@ -0,0 +1,240 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package commands
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/openpubkey/openpubkey/pktoken"
+	"golang.org/x/crypto/ssh"
+)
+
+type InspectCmd struct {
+	// KeyOrCert is the SSH key or certificate to be inspected.
+	KeyOrCert string
+	// Output is where output should be written to.
+	Output io.Writer
```

Comment:
Nice pattern, I need to remember to port this to other the other commands to make getting output in tests easier.